### PR TITLE
Add Hariket Sukeshkumar Sheth to the list

### PR DESCRIPTION
**Pull request by hariketsheth**

### CODE REVIEW

File: public/data/hariketsheth.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      "url": "https://github.com/hariketsheth",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://mobile.twitter.com/HariketSheth",
+      "icon": "twitter"
+    }
+
+
```
Comment (Panquesito7) [inline public/data/hariketsheth.json:17]:
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/hariketsheth.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    },
+    {
+      "title": "Contributor in Girlscript Winter of Contributing",
+      "date": "Sep 2021",
+      "icon": "girlscript",
+      "color": "orange",
+      "description": "Contributor at Girlscript Winter of Contributing 🏆🎖",
+      "url": "https://drive.google.com/file/d/1mbbXHxFbz5x5CD-9BCc_099mBAWJB5uR/preview"
+    }
+
```
Comment (rohan-kulkarni-25) [inline public/data/hariketsheth.json:34]: Can Delete this blank line

Comment (hariketsheth) [reply public/data/hariketsheth.json]: > Can Delete this blank line

I have done the requested edits 👍🏻 .
@rohan-kulkarni-25

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**